public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/networkInsightsDashboard.json

```diff
@@ -0,0 +1,2094 @@
+{
+    "__inputs": [
+    ],
+    "__elements": [],
+    "__requires": [
+      {
+        "type": "grafana",
+        "id": "grafana",
+        "name": "Grafana",
+        "version": "8.5.1"
+      },
+      {
+        "type": "datasource",
+        "id": "grafana-azure-monitor-datasource",
+        "name": "Azure Monitor",
+        "version": "1.0.0"
+      },
+      {
+        "type": "panel",
+        "id": "table",
+        "name": "Table",
+        "version": ""
+      },
+      {
+        "type": "panel",
+        "id": "timeseries",
+        "name": "Time series",
+        "version": ""
+      }
+    ],
+    "editable": true,
+    "id": null,
+    "links": [],
+    "liveNow": false,
+    "panels": [
+      {
+        "collapsed": true,
```

Comment:
We decided to collapse them by default because the same resource group may not have resources available in all types (front door vs load balancer vs app gateway etc). When there are no resources available for the resource type, then the panels that pertain to that resource type will not have any data. 

This leads to a bad user experience, so instead we are letting the user first pick the resource they'd like to visualize and then expand the rows accordingly

Reply:
Thanks for the explanation 👍